draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -460,13 +460,13 @@
 
 
         <t>To overcome the described limitation, and allow a list of footprint constraints that
```

Comment:
I think the "," is redundant. "To overcome the described limitation and allow ...".